docs/source/Synthesis.ipynb

```diff
@@ -0,0 +1,180 @@
+    "This is due to several factors\n",
+    "\n",
+    "- According to the Gottesman-Knill theorem, they can be simulated in polynomial time and space on classical computers using the *stabilizer* formalism.\n",
+    "- They can be used to describe several quantum phenomena such as superposition, entanglement, superdense coding, and teleportation.\n",
+    "- Many error correcting codes rely on them.\n",
+    "\n",
+    "Quantum states that can be obtained from the all-zero basis state $|0\\dots 0\\rangle$ by applying Clifford operations are called stabilizer states.\n",
+    "The name originates from the fact that such a state is uniquely and efficiently described by the set of operators that generate the group of its stabilizers.\n",
+    "Specifically, any *n*-qubit stabilizer state can be described by a set of *n* Pauli strings $\\pm P_{i,0}P_{i,1}P_{i,2}\\dots P_{i,n-1}$, with $P_{i,j}\\in\\{I, X, Y, Z\\}$ and $i, j\\in 0,\\dots, n-1$.\n",
+    "Hence,two bits per qubit are needed to identify the Pauli operator, as well as one additional bit for the phase, which leads to a total of $n(2n+1)` bits needed to uniquely describe a particular stabilizer state.\n",
+    "\n",
+    "The stabilizer representation of a quantum state is conveniently described by a *tableau*:\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "    \\begin{bmatrix}\n",
+    "        x_{0,0}   & \\cdots & x_{0,n-1}   & z_{0,0}    & \\cdots & z_{0,n-1}   & r_0    \\\\\n",
+    "        \\vdots    & \\ddots &  \\vdots         & \\vdots     & \\ddots &    \\vdots         & \\vdots \\\\\n",
+    "        x_{n-1,0} & \\cdots & x_{n-1,n-1} & z_{n-1,0} & \\cdots & z_{n-1,n-1} & r_{n-1}  \\\\\n",
+    "    \\end{bmatrix}\n",
+    "$$\n",
+    "\n",
+    "Here, the binary variables $x_{ij}$ and $z_{ij}$ specify whether the Pauli term $P_{i,j}$ is $X$ or $Z$, respectively.\n",
+    "Since $Y = iXZ$, setting $x_{ij} = z_{ij} = 1$ corresponds to $P_{i,j}=Y$.\n",
+    "Finally, $r_i$ describes whether the generator has a negative phase.\n",
+    "\n",
+    "Consider the following quantum circuit:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.h(0)\n",
+    "qc.cx(0, 1)\n",
+    "qc.h(0)\n",
+    "qc.h(1)\n",
+    "\n",
+    "print(qc.draw(fold=-1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Then, the corresponding stabilizer tableau is\n",
+    "\n",
+    "        0 0 | 1 1 | 0\n",
+    "        1 1 | 0 0 | 0\n",
+    "\n",
+    "which corresponds to the stabilizers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stabilizers = [\"+XX\", \"+ZZ\"]"
```

Comment:
Technically, it should be `[+ZZ, +XX]` here, right? And, in turn, down below. This is also apparent from the two circuits that are produced being different.